modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -411,6 +417,23 @@ private void _validateParametersUnicodeProperties(
 				_objectDefinitionPersistence.fetchByPrimaryKey(
 					objectDefinitionId);
 
+			if (parametersUnicodeProperties.containsKey(
+					"objectDefinitionExternalReferenceCode")) {
+
+				String objectDefinitionExternalReferenceCode =
+					GetterUtil.getString(
+						parametersUnicodeProperties.remove(
+							"objectDefinitionExternalReferenceCode"));
+
+				objectDefinition = _objectDefinitionPersistence.fetchByERC_C(
+					objectDefinitionExternalReferenceCode, companyId);
+
+				objectDefinitionId = objectDefinition.getObjectDefinitionId();
```

Comment:
This line can throw a NullPointer exception :confused:

Reply:
I see that the commit message is correct regarding the behavior, but I can't see reflected in the code.